.github/workflows/deployment-prod.yml

```diff
@@ -0,0 +1,73 @@
+name: "Deployment prod"
+
+on:
+  push:
+    branches:
+      - "master"
+  schedule:
+    - cron:  '0 0 * * *'
+
+env:
+  doctrine_website_github_http_token: "${{ github.token }}"
+  doctrine_website_mysql_password: "${{ secrets.doctrine_website_mysql_password }}"
+  doctrine_website_algolia_admin_api_key: "${{ secrets.doctrine_website_algolia_admin_api_key }}"
+  doctrine_website_stripe_secret_key: "${{ secrets.doctrine_website_stripe_secret_key }}"
+  doctrine_website_send_grid_api_key: "${{ secrets.doctrine_website_send_grid_api_key }}"
+
+jobs:
+  build:
+
+    runs-on: "ubuntu-18.04"
+
+    services:
+      mysql:
+        image: "mysql:5.7"
+        env:
+          MYSQL_DATABASE: "doctrine_website_test"
+          MYSQL_ROOT_PASSWORD: "${{ secrets.doctrine_website_mysql_password }}"
+        ports:
+          - "3306:3306"
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+
+    steps:
+    - uses: "actions/checkout@v2"
+
+    - name: "Cache Composer packages"
+      id: "composer-cache"
+      uses: "actions/cache@v2"
+      with:
+        path: "vendor"
+        key: "${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}"
+        restore-keys: |
+          ${{ runner.os }}-php-
+    - name: "Setup PHP Action"
+      uses: "shivammathur/setup-php@v2"
+      with:
+        php-version: "7.2"
+
+    - name: "Install dependencies"
+      if: "steps.composer-cache.outputs.cache-hit != 'true'"
+      run: "composer install --no-progress --no-suggest --no-dev"
```

Comment:
You could use `ramsey/composer-install` here, it handles the cache part (so you could drop lines 35-42: https://github.com/doctrine/.github/blob/14d0040959e328daf3f0b34960ae84b0fc60bda5/workflow-templates/coding-standards.yml#L33-L36